ramips: add TP-LINK TL-WR902AC v4

kjonca opened the pull request:
Added TP-Link TL0WR90AC v4, by copying v3 and modifying needed modules.

raenye commented on target/linux/ramips/dts/mt7628an_tplink_tl-wr902ac-v4.dts:
```diff
@@ -0,0 +1,99 @@
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
```
Perhaps use the new notation (e.g. button-reset)?

raenye commented on target/linux/ramips/dts/mt7628an_tplink_tl-wr902ac-v4.dts:
```diff
@@ -0,0 +1,99 @@
+	leds {
+		compatible = "gpio-leds";
+
+		lan {
```
Perhaps use the new notation (e.g. led-X).

raenye commented on target/linux/ramips/dts/mt7628an_tplink_tl-wr902ac-v4.dts:
```diff
@@ -0,0 +1,99 @@
+		compatible = "gpio-leds";
+
+		lan {
+			label = "green:lan";
```
Perhaps use `color` and `function` properties instead of `label`.

kjonca replied:
> Perhaps use `color` and `function` properties instead of `label`.

Where can I read about it? Or you want simple textual replacement:
                       color = <LED_COLOR_ID_GREEN>;
                       function = LED_FUNCTION_LAN;
(but still ...)

raenye replied:
Yes, from these the actual label will be derived.
More info: https://elixir.bootlin.com/linux/v5.15.137/source/Documentation/devicetree/bindings/leds/common.yaml

airtonix replied:
@raenye next time to make it easier for people, instead of making them guess what you mean... [just use the suggestion feature](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#:~:text=to%20suggest%20a%20specific%20change%20to%20the%20line%20or%20lines)

raenye replied:
Thanks, wasn't aware of it.

raenye commented:
Please add information about the hardware in the commit message.

kjonca commented:
> Please add information about the hardware in the commit message.

This is not clear for me - what kind of information is expected?

raenye commented:
> This is not clear for me - what kind of information is expected?

See other "add support for" PRs; e.g., #13601 (which is *very* elaborate -- less is also fine).

Commit:
ramips: add TP-LINK TL-WR902AC v4

Seems to be very similar to: https://openwrt.org/toh/tp-link/tl-wr902ac_v3
1 x usb
1 x eth
Powered by mini usb port.
Can use the same TFTP method to install.

Based on dmesg from already flashed device:

SoC Type: MediaTek MT7628AN ver:1 eco:2
CPU0 revision is: 00019655 (MIPS 24KEc)
Memory: 56028K/65536K available
CPU Clock: 580MHz

GoliathLabs commented:
As described in the test formalities check: Signed-off-by is missing or doesn't match author (should be 'Signed-off-by: Kamil Jońca <kjonca@onet.pl>')

kjonca commented:
> As described in the test formalities check: Signed-off-by is missing or doesn't match author (should be 'Signed-off-by: Kamil Jońca [kjonca@onet.pl](mailto:kjonca@onet.pl)')

Well, how can I see it? In "Formalities test" action I can see only "Error:" without any further description"

Commit:
Merge branch 'openwrt:main' into main

airtonix commented:
why did we close this?

kjonca commented:
I

> why did we close this?

I made some mistakes here and could not deal with git and opened 

https://github.com/openwrt/openwrt/pull/13877
